cmd/argocd-application-controller/commands/argocd_application_controller.go

```diff
@@ -58,6 +58,7 @@ func NewCommand() *cobra.Command {
 		metricsPort                      int
 		metricsCacheExpiration           time.Duration
 		metricsAplicationLabels          []string
+		metricsAplicationRevisions       []string
```

Comment:
Minora typo in the other related variable `metricsAplicationLabels`, maybe fix both of these while we're at it?
```suggestion
		metricsApplicationRevisions       []string
```